styles/main/_message.scss

```diff
@@ -110,7 +110,8 @@
 	}
 
 	// Inputs -------------------------------------------------------------- //
-	input.text {
+	input.text,
+	input.file {
```

Comment:
What is this for?

Reply:
Sorry, no use. That's a leftover from playing around with a file chooser, which is of limited use at best and hence abandoned the idea.